dissect/target/helpers/scrape.py

```diff
@@ -74,22 +74,22 @@ def find_needles(
         last_needle_pos = -1
         last_needle_end = 0
         while True:
-            needle_positions = [
-                (pos, needle)
-                for needle in needles
+            needle_positions = []
```

Comment:
On another note (but a bigger ask), maybe it's interesting to write some benchmarks for this piece like we did here: https://github.com/fox-it/dissect.util/pull/64